client.go

```diff
@@ -853,6 +856,11 @@ func (c *TunaSessionClient) ListenUDP(addrsRe *nkngomobile.StringArray) (*udp.En
 		return nil, errors.New("wallet is empty")
 	}
 
+	err = c.startExits()
+	if err != nil {
+		return nil, err
+	}
+
 	host, portStr, err := net.SplitHostPort(c.listeners[0].Addr().String())
 	if err != nil {
 		return nil, err
```

Comment:
We need to make sure listenUDP is only called once. I would suggest the following change:
- save `udp.NewEncryptUDPConn(conn)`
- return the saved udp conn if it's not nil